Marlin/example_configurations/BQ/Hephestos_2/Configuration.h

```diff
@@ -439,20 +431,15 @@
 
 // @section extruder
 
-/**
- * Prevent extrusion if the temperature is below EXTRUDE_MINTEMP.
- * Add M302 to set the minimum extrusion temperature and/or turn
- * cold extrusion prevention on and off.
- *
- * *** IT IS HIGHLY RECOMMENDED TO LEAVE THIS OPTION ENABLED! ***
- */
+// This option prevents extrusion if the temperature is below EXTRUDE_MINTEMP.
+// It also enables the M302 command to set the minimum extrusion temperature
+// or to allow moving the extruder regardless of the hotend temperature.
+// *** IT IS HIGHLY RECOMMENDED TO LEAVE THIS OPTION ENABLED! ***
```

Comment:
This is a regression. The comment has been reformatted.

Reply:
@thinkyhead I'm using this as reference.

https://github.com/MarlinFirmware/Marlin/blob/3be22fc1fc712e5ccc849087290d9daf1fbc9cbf/Marlin/Configuration.h#L441